titus-server-master/src/main/java/com/netflix/titus/master/appscale/service/DefaultAppScaleManager.java

```diff
@@ -210,6 +225,7 @@ public Completable enterActiveMode() {
     public void shutdown() {
         ObservableExt.safeUnsubscribe(reconcileFinishedJobsSub);
         ObservableExt.safeUnsubscribe(reconcileScalableTargetsSub);
+        ObservableExt.safeUnsubscribe(reconcileAllPendingRequests);
```

Comment:
`safeUnsubscribe` takes multiple arguments, so these three lines can be collapsed to one.